jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/JibPluginConfiguration.java

```diff
@@ -354,6 +362,12 @@ protected void checkJibVersion() throws MojoExecutionException {
    * @return the specified platforms
    */
   List<PlatformParameters> getPlatforms() {
+    String property = getProperty(PropertyNames.FROM_PLATFORMS);
+    if (property != null) {
+      return Stream.of(property.split(","))
```

Comment:
For consistency, let's use `ConfigurationPropertyValidator.parseListProperty(property).map(...)...`.